vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/InputFieldView.kt

```diff
@@ -452,6 +452,55 @@ abstract class InputFieldView @JvmOverloads constructor(
         inputField.isSingleLine = singleLine
     }
 
+    /**
```

Comment:
@Endorf requestFocus()?

Reply:
requestFocus method already exists in View class.

Reply:
requestFocus this method already exists